README.md

```diff
@@ -44,3 +44,23 @@ It is possible to use a `substra-backend` skaffold file as the source for the ne
 ```
 SUBSTRA_TESTS_SKAFFOLD_FILEPATH=$SUBSTRA_SOURCE/substra-backend/skaffold.yaml make test
 ```
+
+# Test design guidelines
+
+When adding or modifying tests, please follow these guidelines.
+
+The tests suite:
+- Must be easy to configure through settings files to define the substra network to target
+- Must be independent from the substra network
+  - Substra network must be started separately
+  - Substra network can be located locally / on the cloud
+  - Substra network can be started through docker-compose or through kubernetes
+- Requires a setup with at least 2 organisations
+- Should be able to run multiple times without restarting the network
```

Comment:
It feels hard to read IMO when it starts with "should". What about: It should be possible to run the test suite multiple times without restarting the network

Reply:
I will do the change. 

I used "should" at the beginning to follow one of your previous comment: each comment starting with "should" or "must".

Reply:
Sorry for the contradicting comments! I realized after you made changes that starting with "should" didn't work so well for all the items.